app/views/devise/passwords/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "Forgot password" -%>
```

Comment:
@dsander yeah, I like that one. Pushed.

Re: `Resend unlock instructions `, perhaps s/instructions/code/ and call it good?